src/main/java/org/killbill/billing/client/KillBillHttpClient.java

```diff
@@ -163,7 +165,7 @@ public Response doPost(final String uri, final Object body, final Multimap<Strin
     }
 
     public <T> T doPost(final String uri, final Object body, final Multimap<String, String> options, final Class<T> clazz) throws KillBillClientException {
-        return doPost(uri, body, options, DEFAULT_HTTP_TIMEOUT_SEC, clazz);
+        return doPost(uri, body, options, this.httpTimeOut, clazz);
```

Comment:
You are correct. I actually changed the value requestTimeout to be in seconds since all of the methods are in seconds.